src/main/java/org/sonar/squidbridge/annotations/AnnotationBasedProfileBuilder.java

```diff
@@ -27,12 +27,13 @@
 import org.sonar.api.utils.ValidationMessages;
 
 /**
+ * @deprecated since 2.7 Profile should be manually created.
+ * 
  * Utility class to build an instance of {@link RulesProfile} based on a list of classes annotated
  * with {@link ActivatedByDefault}. It can be used to implement the <code>createProfile</code>
  * method of {@link ProfileDefinition}.
- * 
- *  @since 2.5
  */
+@Deprecated
 public class AnnotationBasedProfileBuilder {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sslr-squid-bridge%3Asslr-squid-bridge&branch=SSLRSQBR-71&issues=AWHXxlo4af12pdu-6jE6&open=AWHXxlo4af12pdu-6jE6)

[ISSUE_KEY]: AWHXxlo4af12pdu-6jE6